### PR TITLE
Gidk 1135 ledger

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.html [hunk 1]
```diff
@@ -280,7 +280,7 @@ <h3 *ngIf="!openDropDown" (dblclick)="openHeaderDropDown()" class="d-inline-bloc
                                 <ng-container *ngIf="txn.type == 'CREDIT'">
 
                                     <td data-title="Date" class="col-xs-2">
-                                        <input class="form-control" autocomplete="off" name="datepic" type="text"
+                                        <input id="giddh-datepicker" class="form-control" autocomplete="off" name="datepic" type="text"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.html:283]: Change ID, it's Duplicate

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

**Review by prateek-giddh — CHANGES_REQUESTED** [review]
Please revert the package-lock changes

File: apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.html (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.html]: Done

Comment (shalinee-walkover) [reply apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.html]: Package-lock JSON added on local but not reflecting on PR

**Pull request closed without merging**